actions/java-gradle-build-jib/action.yaml

```diff
@@ -65,20 +75,41 @@ runs:
         gradle-cache: ${{ inputs.gradle-cache }}
         gradle-cache-read-only: ${{ inputs.gradle-cache-read-only }}
 
+    - name: Process inputs
+      id: process-inputs
+      run: |
+        # Extract the image name from the full image name
+        echo "IMAGE_NAME=$(echo "${{ inputs.full-image-name }}" | cut -d ':' -f 1)" >> "$GITHUB_OUTPUT"
+        IFS=',' read -r -a tags <<< "${{ inputs.additional-tags }}"
+        echo "IMAGE_TAGS<<EOF" >> "$GITHUB_OUTPUT"
+        for element in "${tags[@]}"
+        do
+            echo "type=raw,$element" >> $GITHUB_OUTPUT
+        done
+        echo "EOF" >> "$GITHUB_OUTPUT"
+      shell: bash
+
+    - name: Docker metadata
+      id: meta
+      uses: docker/metadata-action@v5
+      with:
+        images: ${{ steps.process-inputs.outputs.IMAGE_NAME }}
+        tags: |
+          type=ref,event=branch
+          type=sha
+          ${{ steps.process-inputs.outputs.IMAGE_TAGS }}
```

Comment:
Additional tags are a nice feature (which we could also allow in normal Docker image builds in the future), but otherwise why don't we use the same rules for tagging here as [we do for normal Docker images](https://github.com/bakdata/ci-templates/blob/de9f2af69ed0b844280e42004a5f6494981c5dfe/.github/workflows/docker-build-and-publish.yaml#L109-L114) by default? On all branches including the default branch the tag should be `pipeline-${{ github.run_id }}-git-{{ sha }}` and on GitHub tags, the Docker image should be tagged with latest semantic version.

Reply:
Maybe the default tag can even be set as the default for one of the inputs, just like it was done [here](https://github.com/bakdata/ci-templates/blob/de9f2af69ed0b844280e42004a5f6494981c5dfe/.github/workflows/docker-build-and-publish.yaml#L40) before. This allows us to easily have a default defined globally while still allowing changes to it.